usr/share/rear/prep/BACULA/default/400_prep_bacula.sh

```diff
@@ -1,6 +1,19 @@
 ### prepare stuff for BACULA
+if ! test -d "$BACULA_CONF_DIR" ; then
+    test -d "/etc/bacula" && BACULA_CONF_DIR="/etc/bacula"
+    # bacula-enterprise-client uses /opt/bacula/etc
+    test -d "/opt/bacula/etc" && BACULA_CONFIG_DIR="/opt/bacula/etc"
```

Comment:
I think it should be also `BACULA_CONF_DIR` here and not `BACULA_CONFIF_DIR`

Reply:
Indeed - good catch!